components/core/CarouselSidebarData.js

```diff
@@ -458,6 +466,23 @@ class CarouselSidebarData extends React.Component {
     }
   };
 
+  _handleTagDelete = async (tag) => {
+    const response = await Actions.deleteTag({ tag });
+
+    if (response.success) {
+      this.setState({ suggestions: response.tags });
+    }
+
+    if (Events.hasError(response)) {
+      return;
+    }
+  };
+
+  fetchSuggestions = async () => {
+    const res = await Actions.getTagsByUserId();
```

Comment:
Actually, I think this is loading forever for me and never finishing. Not sure why. I just see the loader spinner. Also would be nice to have an option to add a tag by typing even before the list of tags loads

Reply:
Now, you can press "Enter" key to add a tag